_editorial_policies/01_bylaws.md

```diff
@@ -59,6 +61,7 @@ of the members of the editorial board, and who have a history of contributions t
 	    1. All members on the Editorial Board shall be selected in a fair, impartial, and confidential process without regard to race, nationality, religion, gender or gender identity, sexual orientation, or familial status.
          1. Managing editors will be selected based on:
             1. all criteria outlined for associate editors, plus demonstrated excellence serving as an associate editor.
+	 1. The Recording Secretary will be nominated by the Managing editors from among the associate editors.
```

Comment:
"nominated" -> "selected"? Or is there a vote after the nomination?

Reply:
I'm okay with either a selected or nominated/then voted on secretary. Given the general democracy espoused by that section, I wrote it with an implied vote.